frontend/src/components.css

```
@@ -0,0 +1,2546 @@
+}
+
+.tt-page-title {
+  font-size: 24px;
+  font-weight: 700;
+  letter-spacing: -0.5px;
+  color: var(--t1);
+  margin: 0 0 4px;
+}
+
+.tt-page-key {
+  font-family: ui-monospace, SF Mono, Menlo, monospace;
```

Stylelint (17.5.0)

[error] Expected quotes around "SF Mono" (font-family-name-quotes)

(font-family-name-quotes)